src/sys/epoll.rs

```diff
@@ -80,6 +88,13 @@ pub fn epoll_create() -> Result<RawFd> {
 }
 
 #[inline]
+pub fn epoll_create1(flags: EpollFdFlag) -> Result<RawFd> {
+    let res = unsafe { ffi::epoll_create1(flags.bits() | EPOLL_CLOEXEC.bits) };
```

Comment:
In Rust's standard library close-on-exec flag is set by default.
I'm not sure why so, however, I think it's security reason.
Thugh, someone might be surpised when I force to using `fcntl(fd, F_SETFD, !O_CLOEXEC)` if he want.
Okay, I'll fix it.